lib/copy.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * copy.js
+ * https://github.com/skalio/teambeamjs
+ *
+ * Copyright (c) 2015 Skalio GmbH
+ * Licensed under the MIT license.
+ * 
+ * @license
+ */
+
+/* global process */
+
+'use strict';
+
+var Q = require('q'),
+	path = require('path'),
+	osHomedir = require('os-homedir'),
+	mkdirp = require('mkdirp'),
+	Transfer = require('./transfer'),
+	settings = require('./settings'),
+	logHelper = require('./logHelper');
+
+exports.doit = function(options) {
+	logHelper.consoleOutput('Copying transfers for "' + process.principal.email + '"');
+	/*
+	 * - get list of all transfers
+	 * - download files
+	 */
+	var transfersLength = 0;
+	options = options || {};
+	
+	options.dir = options.dir || path.join(osHomedir(), 'Downloads', 'transfers');
+
+	var driveFolderIdx = options['drive'];
```

Comment:
Suggestions:

- use `options.drive` instead of `options['drive']`?
- validate that `options.drive` exists, is numeric